src/natcap/invest/crop_production_percentile.py

```diff
@@ -103,9 +102,23 @@
 }
 
 MODEL_SPEC = {
-    "model_name": MODEL_METADATA["crop_production_percentile"].model_title,
-    "pyname": MODEL_METADATA["crop_production_percentile"].pyname,
-    "userguide": MODEL_METADATA["crop_production_percentile"].userguide,
+    "model_id": "crop_production_percentile",
+    "model_name": gettext("Crop Production: Percentile"),
+    "pyname": "natcap.invest.crop_production_percentile",
+    "userguide": "crop_production.html",
+    "aliases": ("cpp",),
+    "ui_spec": {
+        "order": [
+            ['workspace_dir', 'results_suffix'],
+            ['model_data_path', 'landcover_raster_path', 'landcover_to_crop_table_path', 'aggregate_polygon_path']
+        ],
+        "hidden": ["n_workers"],
+        "sampledata": {
+            "filename": "CropProduction.zip",
```

Comment:
I felt like this was okay because the DataDownloadModal lists out the model names, not the file names, it doesn't appear redundant. The zip file would be downloaded twice if you select both crop production models, but you end up with only one copy of it.

In the longer term, I think it would make more sense to have a 1:1 mapping of sample data to models: https://github.com/natcap/invest/issues/920